docs/how-to/cluster-cfg.rst

```diff
@@ -0,0 +1,50 @@
+Configure cluster_network/ public_network in Microceph
+======================================================
+
+The microceph cluster configuration CLI supports, setting, getting, resetting and listing supported config keys (currently 'cluster_network' and 'public_network'). It is designed to be consistent and intuitive.
+
```

Comment:
Suggest adding a small overview on the concrete task, something like

> To configure the cluster network, the `cluster_network` config key needs to be set to the desired CIDR. Similarly, for configuring the public network, set the `public_network` config key to a CIDR

Reply:
Placing it as plain text felt a bit out of place, so I added a table with key name a description expressing the exact same thing.